src/components/SpectralProfiler/SpectralProfilerComponent.tsx

```diff
@@ -499,10 +499,12 @@ export class SpectralProfilerComponent extends React.Component<WidgetProps> {
             className += " dark-theme";
         }
 
-        let infoHeight = "28px";
-        if (this.plotData?.numProfiles > 1 && this.plotData?.numProfiles <= 5) {
-            infoHeight = `${this.plotData?.numProfiles * 20}px`;
-        } else if (this.plotData?.numProfiles > 5) {
+        const infoMinHeight = "28px";
+        const numProfiles = this.plotData?.numProfiles;
+        let infoHeight = infoMinHeight;
```

Comment:
we don't need to use strings for this. We can just pass a `number` in as `defaultSize` or `minSize`: https://github.com/tomkp/react-split-pane